spec/awesome_spawn_spec.rb

```diff
@@ -31,25 +31,25 @@
       expect(orig_params).to eq(params)
     end
 
-    it "warns about option :in" do
+    it "errors on option :in" do
```

Comment:
FYI:  I just added this in as a separate commit because it was bugging me, but I can drop it if we feel it doesn't have a place here.